acbuild/end.go

```diff
@@ -44,16 +46,13 @@ func runEnd(cmd *cobra.Command, args []string) (exit int) {
 		cmd.Usage()
 		return 1
 	}
-	if len(args) > 1 {
-		stderr("end: incorrect number of arguments")
-		return 1
-	}
 
 	if debug {
 		stderr("Ending build. Writing completed ACI to %s", args[0])
 	}
 
-	err := lib.End(tmpacipath(), args[0], path.Join(contextpath, workprefix), overwrite)
+	err := lib.End(tmpacipath(), args[0], path.Join(contextpath, workprefix),
+		overwrite, sign, args[1:])
```

Comment:
Your use of args[1:] is magical and undocumented, is that intentional? Do we need to expose this to users yet?